geocity/apps/core/static/js/messages_timer.js

```diff
@@ -0,0 +1,8 @@
+// Get all messages of the alert-success class
```

Comment:
@nicolas-heigvd can't you control this in the https://docs.djangoproject.com/en/5.0/ref/contrib/messages/ ?